docs/rackhd/task_annotation.rst

```diff
@@ -0,0 +1,19 @@
+The RackHD Task Annotation is a web UI documentation for RackHD tasks.
+
+The UI page leverage the template of `apidoc <https://github.com/apidoc/apidoc>`_ to display the auto generated task_doc_data.json.
```

Comment:
I don't think it's good to write this in user guide, this is belongs to the design detail.